sys/include/net/ieee802154/radio.h

```diff
@@ -1056,14 +1054,14 @@ static inline bool ieee802154_radio_has_auto_csma(ieee802154_dev_t *dev)
  */
 static inline bool ieee802154_radio_has_sub_ghz(ieee802154_dev_t *dev)
 {
-    return dev->driver->get_cap(dev, IEEE802154_CAP_SUB_GHZ);
+    return (dev->driver->caps & IEEE802154_CAP_SUB_GHZ) != 0;
```

Comment:
for all these functions, it should be enough to do:

```suggestion
    return (dev->driver->caps & IEEE802154_CAP_SUB_GHZ);
```